packages/framework/build.xml

```diff
@@ -1056,7 +1064,7 @@
         </phingcall>
     </target>
 
-    <target name="tests" depends="npm-install-dependencies,error-pages-generate,test-db-demo,test-elasticsearch-index-recreate,test-elasticsearch-export,tests-unit,tests-functional,tests-smoke" description="Runs unit, functional and smoke tests. Builds new test database in the process."/>
+    <target name="tests" depends="npm-install-dependencies,test-db-demo,test-error-pages-generate,test-elasticsearch-index-recreate,test-elasticsearch-export,tests-unit,tests-functional,tests-smoke" description="Runs unit, functional and smoke tests. Builds new test database in the process."/>
```

Comment:
When you now run the whole build (`build-demo-dev`), you will end with generated error pages from test DB, or?

Reply:
Each environment has its own folder for storing generated HTML files